Monika After Story/game/updates.rpy

```diff
@@ -375,7 +375,13 @@ label v0_3_1(version=version): # 0.3.1
 # 0.11.7
 label v0_11_7(version="v0_11_7"):
     python:
-        pass
+        # unlock transgender topic if either gender topics have been shown
+        if (
+                mas_getEVL_shown_count("mas_gender") > 0
+                or mas_getEVL_shown_count("monika_gender_redo") > 0
+        ):
+            mas_unlockEVL("mas_transgender_reveal", "EVE")
+
     return
 
 # 0.11.6
```

Comment:
Hey, fellow tgirl who wants to help with this topic, but I just don't understand what's going on, you created the new topic, but you're leaving the old gender-redo pretty much untouched, expect removing the trans option. What is the plan? It's somehow strange that someone would have to come out as trans in one topic, and then they would have to go to another topic for Moni to actually change pronouns. The name topic should also be linked to it.

Reply:
I'm actually not transgender. My goal was just to create something other people could work off of since this PR has been untouched for so long. I do want this PR to stay away from any transphobia! So if you have any complaints about the dialogue feel free to fix them. And you're right, I'm just one of the devs who isn't as familiar with persistent changes and I didn't want to create something that just wouldn't work. Thanks for your comment, I'll try my best to fix those things.

Reply:
sounds like we should add the option to adjust pronouns/name here then. @kkrosie123 add a todo comment where you think this should be and I'll add the appropriate code.

Reply:
Sounds good!

Reply:
I think there is a problem with the fact that the topic "I want to tell you I am transgender" may mean 2 things:
option 1: someone who already knows their true gender downloads MAS, tells Moni their true gender and then just wants to notify her that they're trans
option 2: someone downloads MAS while still thinking they're cis and later realizes that they're different gender and wants to tell Moni they're trans and update their pronouns and name (note: trans person may still decide to go by their birthname, f.ex. Alex is gender neutral, and some people may figure out they're trans, but still use their birthname for some time because they have troubles choosing new name).

So to clear any confusion, I think at the beginning of the topic Moni should ask what the player means, and then it could be split into the quick section for option 1 and the longer section that would redirect player to pronouns change and optional name change topic for the 2nd option.